zbus/src/connection/builder.rs

```diff
@@ -260,12 +270,16 @@ impl<'a> Builder<'a> {
         self
     }
 
-    /// Register a D-Bus [`Interface`] to be served at a given path.
+    /// Register a D-Bus [`Interface`] `iface` to be served at a given path.
```

Comment:
Actually this is not what I meant. I would never ask you to add unrelated changes to a commit. :) 

I was mistakenly thinking the `iface` attribute is also using `TryFrom` and is therefore fallible. Hence why I asked for it to be also included in the error docs you're adding. 

So I'm sorry for misguiding you but please remove this change.

Reply:
Oh, I was a bit confused about this bit, but it has been a few days too :sweat_smile: 

Should be okay now?

Reply:
@WhyNotHugo yes, thanks. :)